engine/src/main/java/org/terasology/rendering/dag/nodes/CopyToVRFrameBuffersNode.java

```diff
@@ -0,0 +1,118 @@
+import org.terasology.rendering.world.WorldRenderer.RenderingStage;
+import static org.terasology.rendering.opengl.OpenGLUtils.renderFullscreenQuad;
+import static org.terasology.rendering.opengl.ScalingFactors.FULL_SCALE;
+
+public class CopyToVRFrameBuffersNode extends ConditionDependentNode {
+    private static final ResourceUrn LEFT_EYE_FBO = new ResourceUrn("engine:leftEye");
+    private static final ResourceUrn RIGHT_EYE_FBO = new ResourceUrn("engine:rightEye");
+    // TODO: make these configurable options
+
+    @In
+    private OpenVRProvider vrProvider;
+
+    @In
+    private WorldRenderer worldRenderer;
+
+    @In
+    private Config config;
+
+    @In
+    private ScreenGrabber screenGrabber;
+
+    @In
+    private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
+
+    private RenderingConfig renderingConfig;
+    private Material finalPost;
+    private FBO leftEye;
+    private FBO rightEye;
+
+    @Override
+    public void initialise() {
+        renderingConfig = config.getRendering();
+        finalPost = worldRenderer.getMaterial("engine:prog.post");
+        requiresCondition(() -> renderingConfig.isVrSupport());
+        leftEye = requiresFBO(new FBOConfig(LEFT_EYE_FBO, FULL_SCALE,
+                FBO.Type.DEFAULT).useDepthBuffer(),displayResolutionDependentFBOs);
+        rightEye = requiresFBO(new FBOConfig(RIGHT_EYE_FBO,FULL_SCALE,
+                FBO.Type.DEFAULT).useDepthBuffer(),displayResolutionDependentFBOs);
+        if (this.vrProvider != null) {
+            logger.info("Left eye FBOID:" + Integer.toString(leftEye.fboId));
+            logger.info("Right eye FBOID:" + Integer.toString(leftEye.fboId));
+            logger.info("OpenVR init done.");
+            vrProvider.texType[0].handle = leftEye.colorBufferTextureId;
+            vrProvider.texType[0].eColorSpace = JOpenVRLibrary.EColorSpace.EColorSpace_ColorSpace_Gamma;
+            vrProvider.texType[0].eType = JOpenVRLibrary.EGraphicsAPIConvention.EGraphicsAPIConvention_API_OpenGL;
+            vrProvider.texType[0].write();
+            vrProvider.texType[1].handle = rightEye.colorBufferTextureId;
+            vrProvider.texType[1].eColorSpace = JOpenVRLibrary.EColorSpace.EColorSpace_ColorSpace_Gamma;
+            vrProvider.texType[1].eType = JOpenVRLibrary.EGraphicsAPIConvention.EGraphicsAPIConvention_API_OpenGL;
+            vrProvider.texType[1].write();
+        }
+    }
+
+    @Override
+    public void process() {
+        PerformanceMonitor.startActivity("rendering/copyToVRFrameBuffers");
+        logger.info("Process");
+        finalPost.enable();
+        if (!renderingConfig.isVrSupport()) {
+            logger.warn("CopyToVRFrameBufferNode processed in non-VR mode.");
+        } else {
+            renderFinalStereoImage(worldRenderer.getCurrentRenderStage());
+        }
+        PerformanceMonitor.endActivity();
+    }
+
+    private void renderFinalStereoImage(RenderingStage renderingStage) {
```

Comment:
I did think about that actually and I suggested the split-screen mostly to keep in line with the visual behavior we had with the Oculus support. 

If @Cervator has nothing against it I'd be happy with the screen showing only one eye and I'd ask you to open an issue requesting the functionality to _optionally_ see on screen what the HMD displays, including distortion, for demo and debug purposes.